src/color_utils.jl

```diff
@@ -60,12 +60,20 @@ function generate_colorgradient(bgcolor = plot_color(:white);
 	gradient_from_list(colors)
 end
 
-function get_color_palette(palette, bgcolor::Colorant, numcolors::Integer)
+function get_color_palette(palette::Symbol, bgcolor::Colorant, numcolors::Integer)
 	grad = if palette == :auto
 		generate_colorgradient(bgcolor)
 	else
 		cgrad(palette)
 	end
+	if is_categorical(palette)
```

Comment:
It seems a bit strange here that `numcolors` is ignored for categorical palettes. Maybe instead give the first `numcolors` colors, and cycle if `numcolors` is longer (possibly with a warning?).

Reply:
I haven't given it much thought, I was just mimicing the behaviour of the `::Vector{Colorant}` method.
> and cycle if numcolors is longer

Plots already does that automatically.